staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/controller.go

```diff
@@ -133,10 +133,12 @@ func (c *AggregationController) processNextWorkItem() bool {
 }
 
 func (c *AggregationController) sync(key string) (syncAction, error) {
-	err := c.openAPIAggregationManager.UpdateAPIServiceSpec(key)
-	switch {
-	case err != nil:
-		return syncRequeueRateLimited, err
+	if err := c.openAPIAggregationManager.UpdateAPIServiceSpec(key); err != nil {
+		if err == aggregator.ErrAPIServiceNotFound {
+			return syncNothing, nil
+		} else {
+			return syncRequeueRateLimited, err
+		}
```

Comment:
```suggestion
		if err == aggregator.ErrAPIServiceNotFound {
			return syncNothing, nil
		}
		return syncRequeueRateLimited, err
```

Reply:
If we care so much about not found, maybe we should make it explicit in the signature? `found bool, err error`

Reply:
ditto, this was to be consistent with https://github.com/kubernetes/kubernetes/blob/1144c85a929bb7a7d30e3343c660286511afb1e3/staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go#L119-L125
Should we change them together or keep it as is?

Reply:
updated